docs/guides/migrating-from-camunda-platform-7/adjusting-bpmn-models.md

```diff
@@ -0,0 +1,199 @@
+---
+id: adjusting-bpmn-models
+title: Adjusting BPMN models
+description: "Learn how to adjust your BPMN models when migrating from Camunda Platform 7 to Camunda Platform 8."
+---
+
+Ensure your BPMN process models are adjusted as follows to migrate them from Camunda Platform 7 to Camunda Platform 8:
+
+- The namespace of extensions has changed from `http://camunda.org/schema/1.0/bpmn` to `http://camunda.org/schema/zeebe/1.0`.
+- Different configuration attributes are used
+- Camunda Platform 8 has a _different coverage_ of BPMN elements (see [Camunda Platform 8 BPMN coverage](/components/modeler/bpmn/bpmn-coverage.md) versus [Camunda Platform 7 BPMN coverage](https://docs.camunda.org/manual/latest/reference/bpmn20/)), which might require some model changes. Note that the coverage of Camunda Platform 8 will increase over time.
+
+The following sections describe what the existing community-supported [backend diagram converter](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/backend-diagram-converter) does by BPMN symbol and explain unsupported attributes.
```

Comment:
I'm having a difficult time understanding this sentence. Specifically, "backend diagram converter does by BPMN symbol and explain unsupported attributes."

Reply:
I agree - this is a bit awkward. The page is grouped or sectioned by BPMN element and includes information on what attributes are unsupported. Let me take another stab at it.

Reply:
> The following sections describe what the existing community-supported [backend diagram converter](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/backend-diagram-converter) for relevant BPMN symbols, including unsupported element attributes that cannot be migrated.

Better? Worse? No change? 😆

Reply:
Much better. How's this:

The following sections describe the capabilities of the existing community-supported [backend diagram converter](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/backend-diagram-converter) for relevant BPMN symbols, including unsupported element attributes that cannot be migrated.